trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    if super.length <= 10
+      super
+    else
+      super[0..3]
```

Comment:
- [ ] We should return 10 characters, not 4. Kindly fix that.

  ```
  super[0..9]
  ```

Reply:
Thank you